.github/workflows/lint.yaml

```diff
@@ -0,0 +1,24 @@
+name: TypeScript Lint
+
+on:
+  push:
+    # branches:
```

Comment:
is this needed ?

Reply:
Good catch. I commented for testing initially, but I'll uncomment now 😄.